tests/test-class-amp-theme-support.php

```diff
@@ -1338,15 +1338,15 @@ public function test_whitelist_layout_in_wp_kses_allowed_html() {
 	/**
 	 * Test AMP_Theme_Support::conditionally_output_header().
 	 *
-	 * @see AMP_Theme_Support::conditionally_output_header()
+	 * @see AMP_Theme_Support::amend_header_image_with_video_header()
 	 */
 	public function conditionally_output_header() {
```

Comment:
This is an error I made in a commit, but this method `conditionally_output_header()` should be `test_ conditionally_output_header()`.

Reply:
Thanks for noting that. Fixed in 97620b9.